compiler/compiler.go

```diff
@@ -3220,7 +3222,7 @@ func (c *Compiler) ExternalInt64AsPtr() error {
 			c.builder.SetInsertPointAtEnd(entryBlock)
 			var callParams []llvm.Value
 			if fnType.ReturnType() == int64Type {
-				return errors.New("todo: i64 return value in exported function")
+				return errors.New("i64 return value in exported functions disallowed by default, use -wasm-abi=generic to override")
```

Comment:
This is still a TODO, to me it wasn't necessary so I didn't implement it.
What about this: `todo: i64 return value in exported function with -wasm-abi=js`. Still the same error message but with a clue what causes it.

Reply:
Ah, I get it now.
Reworded the message
I believe it's not helpful to show "todo" message to a user: they are not the intended audience and will be only baffled by it. "Not yet implemented" is a clearer wording. Also included a documentation link, since this is the message that would welcome a user that tries to compile a code with an i64 return.

It's also possible that this should NEVER be implemented: after all, substituting i64 value with a pointer not an obvious conversion it might be better to let the user to do it themselves (or switch to a i32/f64 type).